examples/debugging/simple_spread/feedforward/decentralised/run_ippo_with_value_norm.py

```diff
@@ -109,6 +110,12 @@ def network_factory(*args: Any, **kwargs: Any) -> Any:
         clip_value=True,
         normalize_target_values=True,
         normalize_observations=True,
+        normalize_axes=[
```

Comment:
TBH, I am a bit worried using the `normalize_axes` is too complicated. Can we consider passing dict keys to select this? Maybe we can chat about this in standup?